blscosi/protocol/protocol.go

```diff
@@ -396,7 +398,9 @@ func (p *BlsCosi) generateSignature(responses ResponseMap) (kyber.Point, *cosi.M
 	}
 
 	// generate personal signature and append to other sigs
-	personalSig, err := bls.Sign(p.suite, p.Private(), p.Msg)
+	msg := p.Msg
+	AddPrefix(&msg)
+	personalSig, err := bls.Sign(p.suite, p.Private(), msg)
```

Comment:
The spec called for signing `sha256(bls | msg)`. That further reduces the attacker's margin for trying to get a certain thing she wants to be signed. Is there a reason you decide not to do that?

Also: If the existing code you are modifying has historical weirdness in it, like "personal signature", you can change it (comment and variable name). We want that code reads like a simple explanation of what's happening and it is hard to see what "personal" has to do with what's happening here. Perhaps `mySig` or `localSig`?

Reply:
* a6b6462f5b7b3738cd3402ffeea9922a07af7a25 uses now "localSig" instead.
* The "not using sha256" is resolved with @ineiti 's comment on slack: "I think `bls.Sign("domain" | msg)` should be enough then." The reason comes from the `bls.Sign(..)` function that already performs a hashing of the message.